src/main/java/org/fundacionjala/sfdc/core/driver/DriverManager.java

```diff
@@ -13,6 +13,7 @@
  * Class that applies Singleton pattern to instance WebDriver only once.
  */
 public final class DriverManager {
+    //private static final Logger LOGGER = LogManager.getLogger(DriverManager.class.getName());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)